app/connectors/VatDeferralNewPaymentSchemeConnector.scala

```diff
@@ -21,11 +21,10 @@ import javax.inject.Inject
 import models.Eligibility
 import play.api.Logging
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
-
 import scala.concurrent.{ExecutionContext, Future}
 
 class VatDeferralNewPaymentSchemeConnector @Inject()(http: HttpClient,
```

Comment:
There is a compilation warning on this class "method readFromJson in trait HttpReadsLegacyJson is deprecated (since 11.0.0): Use uk.gov.hmrc.http.HttpReads.Implicits". It is also interesting to note that this connector is not a singleton unlike the other connectors.